waypoints: [ 
  [37.4506893, 126.652787, 50.0], #left bottm
  [37.451115, 126.656213, 49.0], #mid up
  [35.069481592976622, 128.57889197025276, 49.0],
#   [35.069464319011999, 128.57896821820279, 49.0], #right bottom 
#   [35.069558500130611, 128.57893036813357, 49.0], #right up
#   [35.069387336800794, 128.57896450335267, 49.0] # startpoint
]

goal_range: 1.5

kp_servo : 0.4
ki_servo : 0.0
kd_servo : 0.1

servo_range: [78, 113]

thruster: 1600
